packages/babel-types/src/modifications/appendToMemberExpression.ts

```diff
@@ -1,13 +1,13 @@
-// @flow
 import { memberExpression } from "../builders/generated";
+import type * as t from "../types";
 
 /**
  * Append a node to a member expression.
  */
-export default function appendToMemberExpression<T: Object>(
+export default function appendToMemberExpression<T extends t.MemberExpression>(
   member: T,
-  append: Object,
-  computed?: boolean = false,
+  append: t.MemberExpression["property"],
```

Comment:
I wonder if this should just be

```ts
export default function appendToMemberExpression(
  member: t.MemberExpression,
  append: t.MemberExpression["property"],
  computed: boolean = false,
```

Reply:
yeah, looks reasonable - updating it…

not sure why in flow sources it was made as generic